src/main/java/org/fundacionjala/pivotal/pageobjects/login/SignInPage.java

```diff
@@ -68,14 +68,19 @@ public Dashboard clickSignInButton() {
      * @return .
      */
     public static Dashboard loginAs(final String username, final String password) {
-        WebDriverManager.getInstance().getDriver().get(HomePage.HOME_PAGE_URL);
-        HomePage homePage = new HomePage();
-        SignInPage signInPage = homePage.clickOnSignInButton();
-        signInPage.setUsernameTextBox(username);
-        LOGGER.log(Level.INFO, "Sign in on pivotaltracker.com, user: " + username);
-        signInPage.clickOnLoginButton();
-        signInPage.setPasswordTextBox(password);
-        return signInPage.clickSignInButton();
+        try {
+            WebDriverManager.getInstance().getDriver().get(HomePage.HOME_PAGE_URL);
+            HomePage homePage = new HomePage();
+            SignInPage signInPage = homePage.clickOnSignInButton();
+            signInPage.setUsernameTextBox(username);
+            LOGGER.log(Level.INFO, "Sign in on pivotaltracker.com, user: " + username);
+            signInPage.clickOnLoginButton();
+            signInPage.setPasswordTextBox(password);
+            return signInPage.clickSignInButton();
+        } catch (NullPointerException ex) {
+            LOGGER.error("There isn't a url in the config.properties file");
+            throw new RuntimeException();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)